portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
-*/
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getMessages() {
+  fetch('/data').then(response => response.json()).then((messages) => {
+    const messagesElement = document.getElementById('messages-container');
+    messagesElement.innerHTML = "";
+    console.log(messages);
+    for (i of messages) {
+      messagesElement.innerHTML += "<p>"+i+"</p><br>";
```

Comment:
BTW, Uty in one of his CLs has a really cool way of rendering the list of messages without requiring the use of `innerHTML`. I'd encourage you to ping him or the team chat to discuss this.

Reply:
Uty and Kiba, actually :)

Reply:
Sounds Good!